modules/@apostrophecms/area/ui/apos/components/AposAreaEditor.vue

```diff
@@ -422,9 +422,16 @@ export default {
       } else if (this.widgetIsContextual(name)) {
         return this.insert({
           widget: {
-            _id: cuid(),
             type: name,
-            ...this.contextualWidgetDefaultData(name)
+            ...this.contextualWidgetDefaultData(name) // TODO: remove?
+          },
+          index
+        });
+      } else if (!this.widgetHasInitialModal(name)) {
+        return this.insert({
+          widget: {
+            type: name,
+            aposPlaceholder: true
```

Comment:
Do you mean `this.contextualWidgetDefaultData(name)` if widget is contextal?

Because the `def` implementation described in the tech design is not in the ticket scope, but I can handle it in the PR regardless